mod.ts

```diff
@@ -55,6 +55,7 @@ export default function (
     .ignore("node_modules")
     .ignore("import_map.json")
     .ignore("deno.json")
+    .ignore("deno.jsonc")
     .ignore((path) => path.endsWith("/.DS_Store"))
```

Comment:
By the way, isn't this already ignored? Quoting https://lume.land/docs/configuration/ignore-files/:

> By default, all files and directories starting with `.` or `_` are ignored.

Reply:
Mmm, yes. I think I've added it to prevent to copy static files (with `site.copy("dirname")`) because the copy function only check `ignore()` conditions but doesn't check if the file starts with `.` or `_`. But I agree that this is a bug and should do this check too, so this condition is not needed.